ulog/log.go

```diff
@@ -186,6 +193,16 @@ func (l *baseLogger) fieldsConversion(keyVals ...interface{}) []zap.Field {
 				fields = append(fields, zap.Marshaler(key, value))
 			case fmt.Stringer:
 				fields = append(fields, zap.Stringer(key, value))
+			case stackTracer:
+				stack := value.StackTrace()
+				lines := make([]zap.Field, 0, len(stack))
+				for _, frame := range stack {
+					s := fmt.Sprintf("%s", frame)
+					l := fmt.Sprintf("line:%d", frame)
```

Comment:
I think `file:line` is more like go traces, rather than `file: foo, line: 123`
If that's the case, we can just do an array of strings for trace, thoughts?